wrapper/godot-library/src/main/kotlin/godot/core/Color.kt

```diff
@@ -5,14 +5,95 @@ import godot.gdnative.*
 import kotlinx.cinterop.*
 import kotlin.math.*
 
-class Color(var r: Double, var g: Double, var b: Double, var a: Double = 1.0) : Comparable<Color>, CoreType {
-    constructor() :
-            this(0.0, 0.0, 0.0, 1.0)
+class Color(var r: Double, var g: Double, var b: Double, var a: Double = 1.0): Comparable<Color>, CoreType {
```

Comment:
Just as a note:
Personally I think in the feature we should go with the scout boys approach and leave to code cleaner as we found it. Means: From our predecessors there is some pretty ugly formatting in all classes. So my suggestion is to reformat the whole class when we need to modify anything in it. With this, all the classes will get clean and concise code over time.
What do you think @piiertho ? Just a suggestion that came to my mind looking at this class in general :-) Also you would not have to do it in this pr as it's too important for now :-)